hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -38,4 +38,19 @@ objects:
     channel: ${CHANNEL}
     name: aws-account-operator
     source: aws-account-operator-catalog
-    sourceNamespace: aws-account-operator
\ No newline at end of file
+    sourceNamespace: aws-account-operator
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: aws-account-operator-client
+  subjects:
+  - kind: ServiceAccount
+    namespace: aws-account-operator-client
+    name: aws-account-operator-client
+  - kind: Group
+    name: aws-account-operator-client
+  roleRef:
+    kind: ClusterRole
+    name: aws-account-operator-client
```

Comment:
Where is this ClusterRole being created? it doesn't exist in hive prod currently